docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -25,8 +27,10 @@ To try out Camunda 8 or develop against it, consider signing up for our [SaaS of
 
 While the guide is primarily tailored for UNIX systems, it can also be run under Windows by utilizing the [Windows Subsystem for Linux](https://learn.microsoft.com/windows/wsl/about).
 
-:::warning
+:::warning Cost Management
```

Comment:
just for consistency, you used in the other doc `Cost management`, which generally aligns with the titles.
Doesn't matter too much since the admonition puts it anyway in uppercase when rendered.